src/Reflex/Class.hs

```diff
@@ -296,9 +297,14 @@ class ( MonadHold t (PushM t)
   -- | Construct a 'Coercion' for a 'Dynamic' given an 'Coercion' for its
   -- occurrence type
   dynamicCoercion :: Coercion a b -> Coercion (Dynamic t a) (Dynamic t b)
+  -- | Construct a 'Coercion' for an 'Incremental'' given an 'Coercion' for its
+  -- occurrence type
+  incrementalCoercion :: Coercion t1 t2 -> Coercion p1 p2 -> Coercion (Incremental' t t1 p1) (Incremental' t t2 p2)
```

Comment:
I take it something goes wrong if you do this instead?
```haskell
incrementalCoercion :: Coercion (PatchTarget p1) (PatchTarget p2) -> Coercion p1 p2 -> Coercion (Incremental t p1) (Incremental t p2)
```

Reply:
That's a very good question. That probably will work, and it's much less horrible.